### PR TITLE
changes site class options based on the location that is selected

### PR DESCRIPTION
**Pull request by jdbrown-USGS**
Fixes usgs/earthquake-hazard-tool#56

- Changes selectable site classes based on location.

### CODE REVIEW

**Comment by emartinez-usgs** [comment]
Talked this over and we want this to be a data-driven implementation. Do not change existing data extent, but rather add a new extent to the database and use that for the UI.

**Comment by jdbrown-USGS** [comment]
added new extent to the database and used that for the UI.

File: src/lib/classes/RegionFactory.class.php [hunk 1]
```diff
@@ -19,7 +19,11 @@ protected function _initialize () {
         minlatitude,
         maxlatitude,
         minlongitude,
-        maxlongitude,
+        maxlongitude
```
Comment (emartinez-usgs) [inline src/lib/classes/RegionFactory.class.php]: Missing comma.

File: src/lib/classes/RegionFactory.class.php [hunk 2]
```diff
@@ -38,7 +42,11 @@ protected function _initialize () {
...
```
Comment (emartinez-usgs) [inline src/lib/classes/RegionFactory.class.php]: Missing comma.

**Pull request merged**